### PR TITLE
fix(get_all_tables_with_cdc): Fix removing cdc_log_suffix

### PR DESCRIPTION
**Pull request by aleksbykov**
On step removing cdc_log_suffix to get base table name
incorrectly used function strip(). if keyspace or
table starts or finished with chars from cdc_log_suffix
such chars removed and list contains wrong name of keyspace.table

Instead that, using the slicing

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~New configuration option are added and documented (in `sdcm/sct_config.py`)~
- [ ] ~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~
- [ ] ~All new and existing unit tests passed (CI)~
- [ ] ~I have updated the Readme/doc folder accordingly (if needed)~

### CODE REVIEW

**Comment by aleksbykov** [comment]
Should be backported after backport https://github.com/scylladb/scylla-cluster-tests/pull/2881

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -2632,19 +2634,22 @@ def _alter_table_with_cdc_properties(self, keyspace: str, table: str, cdc_settin
         with self.cluster.cql_connection_patient(self.target_node) as session:
             session.execute(cmd)
         # wait applying cdc configuration
-        time.sleep(15)
+        time.sleep(30)
```
Comment (amoskong) [inline sdcm/nemesis.py:2637]: Why do you reduce the sleep time? Please split the change to another commit

**Review by amoskong — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]
